src/_sass/layout.scss

```diff
@@ -1338,3 +1338,40 @@ input[type=checkbox]:checked + .formItem-checkbox {
 .language-bash code {
   user-select: all;
 }
+.banner {
+  position: absolute;
+  display: flex;
+  align-items: center;
+  top: 0;
+  left: 0;
+  right: 0;
+  padding: $spacing-unit / 4;
+  justify-content: space-around;
```

Comment:
There's some CSS in here that's redundant — artifacts of the iteration.

Reply:
ya, simplified the html/css a bit. some of it is necessary to override the default body styles